Add a link to report an Issue

hi-rustin opened the pull request:
close https://github.com/rust-lang/crates.io/issues/3129

rust-highfive commented:
r? @smarnach

(rust-highfive has picked a reviewer for you, use r? to override)

hi-rustin commented:
![image](https://user-images.githubusercontent.com/29879298/126112260-eed6ac94-87c3-498e-9d60-3d0696ffbb47.png)

I'm not sure we should force a line break?

JohnTitor commented on app/components/footer.hbs:
```diff
@@ -8,10 +8,12 @@
     <span local-class="sep">|</span>
     <a href='https://www.rust-lang.org/policies/security'>Report a security issue</a>
     <span local-class="sep">|</span>
+    <a href='https://github.com/rust-lang/crates.io/issues/new/choose'>Report an issue</a>
```
I couldn't attend the meeting so I have no context, but I think it's clearer to say "Report a general issue", considering we have "Report a security issue". And I'd prefer to place the security link next to the general link.

hi-rustin replied:
Make sense. Addressed. Thanks for your review!

JohnTitor reviewed (approved):
The change itself looks good, though I'm going to defer the final review to another reviewer as I'm not sure if we need a style change.

hi-rustin commented:
PC:
![image](https://user-images.githubusercontent.com/29879298/127768087-c98b5479-51a2-4840-a729-7b39390282db.png)

mobile:
![image](https://user-images.githubusercontent.com/29879298/127768105-bb2cbde8-1994-48f1-bd2b-ca52a70c9379.png)

hi-rustin commented:
r? @jtgeibel

Could you please take a look? Thanks!

hi-rustin commented:
@Turbo87 

Could you please take a look? Thanks!

Turbo87 reviewed (approved):
probably not the final state of the footer, but I guess this is a good intermediate step :)

Turbo87 commented:
@bors r+

bors commented:
:pushpin: Commit ad0fba568e29855dc9cad7156c1cb69b7b38c3bf has been approved by `Turbo87`

<!-- @bors r=Turbo87 ad0fba568e29855dc9cad7156c1cb69b7b38c3bf -->
<!-- homu: {"type":"Approved","sha":"ad0fba568e29855dc9cad7156c1cb69b7b38c3bf","approver":"Turbo87"} -->

bors commented:
:hourglass: Testing commit ad0fba568e29855dc9cad7156c1cb69b7b38c3bf with merge 0435e20962d9b01dc9ec16eecb120f4baf6d8008...
<!-- homu: {"type":"BuildStarted","head_sha":"ad0fba568e29855dc9cad7156c1cb69b7b38c3bf","merge_sha":"0435e20962d9b01dc9ec16eecb120f4baf6d8008"} -->

bors commented:
:sunny: Test successful - [checks-actions](https://github.com/rust-lang/crates.io/runs/3371593709)
Approved by: Turbo87
Pushing 0435e20962d9b01dc9ec16eecb120f4baf6d8008 to master...
<!-- homu: {"type":"BuildCompleted","approved_by":"Turbo87","base_ref":"master","builders":{"checks-actions":"https://github.com/rust-lang/crates.io/runs/3371593709"},"merge_sha":"0435e20962d9b01dc9ec16eecb120f4baf6d8008"} -->

epage commented:
As a user, "report an issue" sounds definitive, that I know there is an issue.  I don't remember what issue led to #3129 but I had written "I was trying to get support with a potential problem with crates.io".  The new link would not have helped me in this case.  I probably would still click "send an email" because (1) it sounds closest to a means of support and (2) there is no clarification on when emailing should be used.

hi-rustin commented:
> As a user, "report an issue" sounds definitive, that I know there is an issue. I don't remember what issue led to #3129 but I had written "I was trying to get support with a potential problem with crates.io". The new link would not have helped me in this case. I probably would still click "send an email" because (1) it sounds closest to a means of support and (2) there is no clarification on when emailing should be used.

The link I added here is `report a general issue
`, maybe we do need a link that explains how to use them. This is not the final footer state, maybe we can consider it in a future design?

Also, I added it just thinking we could jump to here via a direct link. It's also what you mentioned in the possible alternative solutions.
